ruby_on_rails/cucumber.md

```diff
@@ -0,0 +1,86 @@
+# Cucumber
+
+[Cucumber](https://cucumber.io) is a testing framework which allows us to specify feature tests using plain language.
+In our projects, we use Cucumber for e2e tests. We integrate it using the
+`cucumber-rails` gem.
+
+## Installing `cucumber-rails`
+
+The installation process of `cucumber-rails` is also documented on its
+[Github page](https://github.com/cucumber/cucumber-rails).
+
+To install `cucumber-rails`, add the following gems to your Gemfile:
+
+```rb
+group :test do
+    gem 'capybara', require: false
+    gem 'cucumber-rails', require: false
+    gem 'database-cleaner'
+end
+```
+
+Note that `capybara` and `database-cleaner` are optional. Most probably, you are going to need them.
```

Comment:
Why is that? Can you explain me when they would not be needed?

Reply:
👍 In our case (we use Cucumber for e2e tests), they are always needed. I am going to either remove this line or write down why we are using these gems.